### ISSUE TITLE
Hope to provide a plugin for vscode

### ISSUE THREAD

**Issue by wisinfun** [drawdb-io/drawdb#85]
Drawdb can be drawn directly in vscode

**Comment by 1ilit** [comment]
Yes! I'm going to be working on a vscode extension during the summer break

**Comment by jifbrodeur** [comment]
@1ilit Excellent idea, the vscode plugin

**Comment by khushal786** [comment]
@1ilit I can help you around let me know

**Comment by dantefung** [comment]
Looking forward to vscode plug-in

**Comment by Xiaokang2022** [comment]
Sorry to bother, but any updates on this issue?

**Comment by daviddwlee84** [comment]
Looking forward to it!

---

other great integration examples

- [Draw.io Integration - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio)
- [Excalidraw - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=pomdtr.excalidraw-editor)
